dlup/data/dataset.py

```diff
@@ -258,15 +280,19 @@ def __getitem__(self, index):
 
         region = region_view.read_region(coordinates, region_size)
 
-        sample: StandardTilingFromSlideDatasetSample = {
+        sample: TileSample = {
             "image": region,
             "coordinates": coordinates,
             "mpp": mpp,
             "path": self.path,
             "region_index": region_index,
+            "labels": None,
```

Comment:
Should we add these None variables by default? If we then don't have any labels or annotations in the dataset, we will end up with None in our batches, which torch will complain about when collating. Probably adding them only if self.annotations/labels is not None works more smoothly

Reply:
Yes, otherwise you can’t make it a TypedDict